### ISSUE TITLE
Trusty flake8 issues

### ISSUE THREAD

**Issue by panlinux** [CanonicalLtd/ubuntu-advantage-client#242]
From https://launchpadlibrarian.net/417282221/buildlog_ubuntu-trusty-amd64.ubuntu-advantage-tools_19.1-0ubuntu1~354~git.28e603a~ubuntu14.04.1_BUILDING.txt.gz:
```
# required for Trusty
flake8 --ignore=E901 uaclient
uaclient/tests/test_cli_attach.py:23:56: E251 unexpected spaces around keyword / parameter equals
uaclient/tests/test_cli_attach.py:23:58: E251 unexpected spaces around keyword / parameter equals
uaclient/tests/test_cli_attach.py:28:47: E251 unexpected spaces around keyword / parameter equals
uaclient/tests/test_cli_attach.py:28:49: E251 unexpected spaces around keyword / parameter equals
uaclient/tests/test_cli_attach.py:35:44: E251 unexpected spaces around keyword / parameter equals
uaclient/tests/test_cli_attach.py:35:46: E251 unexpected spaces around keyword / parameter equals
uaclient/tests/test_cli_attach.py:42:52: E251 unexpected spaces around keyword / parameter equals
uaclient/tests/test_cli_attach.py:42:54: E251 unexpected spaces around keyword / parameter equals
make[1]: *** [override_dh_auto_test] Error 1
```

**Comment by OddBloke** [comment]
In our `flake8-trusty` tox environment, we run `flake8 uaclient setup.py` which is why we aren't catching these in CI. We should ensure that the two are aligned.

**Comment by panlinux** [comment]
This should be the last FTBFS for trusty, considering we are already taking the package in trusty-proposed.

**Comment by OddBloke** [comment]
I believe this was fixed in #247 (and a further build failure was fixed in #248).
